man/Makefile.am

```diff
@@ -119,7 +119,7 @@ nodist_man_MANS = \
 
 dist_noinst_DATA += $(dist_noinst_man_MANS) $(dist_man_MANS)
 
-SUBSTFILES += $(nodist_man_MANS)
+SUBSTFILES += $(nodist_man_MANS) %D%/man8/zfs_prepare_disk.8
```

Comment:
Maybe this should be put into `nodist_man_MANS`?

Reply:
This problem is separately fixed in my new PR #15979.

Reply:
Yes, indeed.   Let's deal with it in https://github.com/openzfs/zfs/pull/15979.